settings.gradle

```diff
@@ -1 +1,20 @@
-include ':AutomatticTracks'
+pluginManagement {
+    plugins {
+        id 'org.jetbrains.kotlin.android' version '1.4.31'
+    }
+    repositories {
+        google()
+        gradlePluginPortal()
+        jcenter()
+    }
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.id.id.contains("com.android")) {
+                useModule("com.android.tools.build:gradle:4.0.2")
+            }
+        }
```

Comment:
It's required as `AGP` doesn't support [Plugin Markers Artifacts](https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers). [It'll be fixed in AGP 4.2.0 ](https://issuetracker.google.com/issues/64551265#comment10)